ADOPTERS.md

```diff
@@ -19,7 +19,7 @@ This is a list of adopters using PowerShell in production or in their products (
   In addition, Azure Functions provides client tools such as [`Az.Functions`](https://www.powershellgallery.com/packages/Az.Functions), a cross-platform PowerShell module to manage function apps and service plans in the cloud.
   For more information about Functions, please visit [functions overview](https://learn.microsoft.com/azure/azure-functions/functions-overview).
 * [PowerShell Universal](https://ironmansoftware.com/powershell-universal) is a cross-platform web framework for PowerShell.
-  It provides the ability to create robust, interactive websites, REST APIs, and Electron-based desktop apps with PowerShell script.
+  It provides the ability to create robust, interactive web sites, REST APIs, and Electron-based desktop apps with PowerShell script.
```

Comment:
This should stay as `websites`

Reply:
The linter insists it should be `sites` only:
> /github/workspace/ADOPTERS.md
  22:57  ✓ error  Incorrect usage of the term: “web sites”, use “sites” instead  terminology